examples/private_connection/README.md

```diff
@@ -0,0 +1,61 @@
+# Private Connectivity Example
+
+This example shows how to configure a forseti to use only private connection between server, client and GCP services (e.g. Storage or Cloud SQL).
+
+This example deploys the following:
+
+1. Forseti infrastructure
+   * CloudSQL Database
+   * Forseti Server GCS Bucket
+   * Forseti Client GCS Bucket
+   * Forseti Client VM
+   * Forseti Server IAM Service Account
+   * Forseti Client IAM Service Account
+2. Forseti on GKE - forseti-on-gke
+3. DNS and NAT configuration for private communication between forseti server and GCP services
```

Comment:
DNS, firewall and NAT configurations ...

Reply:
done